webpack/scenes/Organizations/OrganizationSelectors.js

```diff
@@ -1,4 +1,18 @@
 export const selectOrganizationState = state => state.katello.organization;
 
+export const selectManifestName = state =>
+  selectOrganizationState(state).owner_details?.upstreamConsumer?.name;
+
+// for use in ManageManifestModal to replace getManifestName()
+export const selectManifestHref = state =>
+  selectManifestName(state) && [
+    'https://',
+    selectOrganizationState(state).owner_details.upstreamConsumer.webUrl,
+    selectOrganizationState(state).owner_details.upstreamConsumer.uuid,
+  ].join('/');
+
+export const selectIsManifestImported = state =>
```

Comment:
It looks like we have two ways of making this determination now. Do you think it should stay that way, or could it be consolidated? 

https://github.com/Katello/katello/blob/ac0620f2d614d6ad690008f3f5a2d7848972ce6e/webpack/scenes/Subscriptions/SubscriptionHelpers.js#L6-L7

Reply:
good catch!  It should probably be consolidated, but I'd say the one in `SubscriptionHelpers.js` is basically a selector, but in the wrong file.  So we should consolidate in favor of `selectIsManifestImported`